modules/ROOT/pages/backup-restore/online-backup.adoc

```diff
@@ -280,6 +263,19 @@ For example, if your current database has a `Total mapped size` of `128GB` as pe
 [[online-backup-resources]]
 === Computational resource configurations
 
+Checkpointing::
```

Comment:
For example, something like this:
```
[[online-backup-timout]]
=== Backup timeout configurations

Full or incremental backups might sometimes fail due to server unavailability, network or local resource issues, or long checkpoints. 
In such cases, you can use the xref:configuration/configuration-settings.adoc#config_dbms.cluster.catchup.client_inactivity_timeout[`dbms.cluster.catchup.client_inactivity_timeout`] configuration setting to extend the time for network inactivity.
It controls the timeout duration of the catchup protocol, which is the underlying protocol of multiple catchup processes, including backups.

If the backup times out due to a long checkpoint, you can also tune up the xref:configuration/configuration-settings/#config_db.checkpoint.iops.limit[db.checkpoint.iops.limit] and ensure your disks are performant enough to handle the load. 
For more information, see xref:performance/disks-ram-and-other-tips/#performance-checkpoint-iops-limit[Checkpoint IOPS limit].
```

Reply:
Of course, we can. But in fact, it's a completely different task. The idea was to stress out the importance of checkpointing to backups success, and you suggest to detail backups timeouts because of server unavailability or general unresponsiveness, local or network resources, and checkpoints.
Probably. It's a good idea. But will it require additional details or clarifications? @MishaDemianenko, @tselmegbaasan, what do you think?